### PR TITLE
Re-enable melee breakdown

### PR DESCRIPTION
**Pull request by reschan**
image

### CODE REVIEW

File: js/sq2display.js [hunk 1]
```diff
@@ -845,6 +826,13 @@ function displaysq2MeleeDamage(parent_elem, overallparent_elem, meleeStats){
     critStats.append(critChance);
 
     parent_elem.append(critStats);
+
+    //up and down arrow - done ugly
+    let arrow = document.createElement("img");
+    arrow.id = "arrow_" + overallparent_elem.id;
+    arrow.style.maxWidth = document.body.clientWidth > 900 ? "3rem" : "10rem";
+    arrow.src = "../media/icons/" + (newIcons ? "new" : "old") + "/toggle_down.png";
+    overallparent_elem.appendChild(arrow);
```
Comment (hppeng-wynn) [inline js/sq2display.js:835]: Can this code be combined with the existing arrow code to reduce duplication? ig just separate function prolly so its standardized if we ever decide to change them

Comment (reschan) [reply js/sq2display.js:835]: Yea should've done that oops

Comment (ferricles) [reply js/sq2display.js:835]: res?

**Comment by ferricles** [comment]
lgtm

**Pull request merged**